File: MiniDOMWebCtrls/DMD/Sistema/CAttachmentsModuleHandler.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Forms
Imports minidom.WebSite
Imports minidom.Databases
Imports minidom.Forms.Utils



Namespace Forms

   
 
    Public Class CAttachmentsModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SExport Or ModuleSupportFlags.SImport)
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New CAttachmentsCursor
        End Function

    End Class
 
    
End Namespace

File: MiniDOMWebCtrls/DMD/Sistema/CAttachmentsModuleHandler.cs

namespace minidom.Forms
{
    public class CAttachmentsModuleHandler : CBaseModuleHandler
    {
        public CAttachmentsModuleHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SEdit | ModuleSupportFlags.SExport | ModuleSupportFlags.SImport)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Sistema.CAttachmentsCursor();
        }
    }
}